src/app/modules/time-clock/components/entry-fields/entry-fields.component.spec.ts

```diff
@@ -167,19 +169,43 @@ describe('EntryFieldsComponent', () => {
     component.setDataToUpdate(entry);
     spyOn(toastrServiceStub, 'error');
 
-    const hourInThePast = moment().subtract(6, 'hour').format('HH:mm:ss');
+    const hourInThePast = moment().subtract(6, 'hour').format('HH:mm');
     component.entryForm.patchValue({ start_hour : hourInThePast});
     component.onUpdateStartHour();
 
     expect(toastrServiceStub.error).toHaveBeenCalled();
+    expect(component.showtimeInbuttons).toEqual(false);
+  });
+
+  it('should show time In buttons when time is modified', () => {
```

Comment:
Maybe changing the message of the test would be more readable, like: "should show time In control buttons when time is modified"

Reply:
Done!